cmd/snap/cmd_run_test.go

```diff
@@ -1899,6 +1899,11 @@ func (s *RunSuite) TestWaitWhileInhibitedGraphicalSessionFlow(c *check.C) {
 	restoreIsGraphicalSession := snaprun.MockIsGraphicalSession(true)
 	defer restoreIsGraphicalSession()
 
+	restoreTryNotifyRefresh := snaprun.MockTryNotifyRefreshViaSnapDesktopIntegrationFlow(func(snapName string) (bool, error) {
```

Comment:
Given that we add mocking here, it seems like a good opportunity to also add checks here, e.g. `c.Check(snapName, Equals, "some-snap")` here and in similar places (assuming it's `some-snap`)